Python/pdf_reorder/README.md

```diff
@@ -0,0 +1,32 @@
+
+## A pdf reorder or Rearranging Script
+
+#### I used the pdfrw library 
+It is built purely in python
+
+A more efficient,faster and non python built alternatives is PyMuPDF.
+The PyMuPDF library can dependency issues hence we will go for a more simpler **pdfrw library
+
+You can download pdfrw using
+	pip install pdfrw
```

Comment:
Use : 

```

```pip install pdfrw```

```

to format the Code or Commands you add on the README